go/cmd/pathwar/main.go

```diff
@@ -648,6 +663,84 @@ func main() {
 
 				fmt.Println("OK")
 
+				return nil
+			},
+		}, {
+			Name:      "challenge-flavor-add",
+			Usage:     "pathwar [global flags] admin [admin flags] challenge-flavor-add [flags] [args...]",
+			ShortHelp: "add a challenge flavor",
+			FlagSet:   adminChallengeFlavorAddFlags,
+			Exec: func(args []string) error {
+				if err := globalPreRun(); err != nil {
+					return err
+				}
+
+				ctx := context.Background()
+				apiClient, err := httpClientFromEnv(ctx)
+				if err != nil {
+					return errcode.TODO.Wrap(err)
+				}
+
+				challengeID, err := strconv.ParseInt(addChallengeFlavorChallengeID, 10, 64)
```

Comment:
you can avoid parsing by using an int flag

Reply:
Thanks !